### DISCUSSION TITLE
Can the apiResource feature be extended to support filtering so we don't have to code the filters each time?

### DISCUSSION THREAD

**Discussion by ghost** [laravel/framework#50243] [Ideas]
Imagine you have to rewrite a monolith that has tens or even hundreds tables. It is nonsense to manually build the filters for all of those tables in a microservices/services architecture (and even if you need them exposed for a FE framework).

**Comment by macropay-solutions** [comment]
We spent almost 3 years building such a feature which we are using in production almost since we started it. We sent emails to Tailor about it but did not get any reply.
Also no positive feedback from the community in regards to this...

UPDATE: we published a free version without filtering:
 https://github.com/laravel/framework/discussions/49577#discussioncomment-8787463
